tibanna/check_task.py

```diff
@@ -77,6 +77,8 @@ def run(self):
         if does_key_exist(bucket_name, job_aborted):
             try:
                 self.handle_postrun_json(bucket_name, jobid, self.input_json, public_read=public_postrun_json)
+                # Instance should already be terminated here. Sending a second signal just in case
+                boto3.client('ec2').terminate_instances(InstanceIds=[instance_id]) 
```

Comment:
I am just adding termination commands here. Since the `check_task` lambda is independent from the actual workflow, that should be good enough in my opinion.